lib/internal/loader/Loader.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+const { URL } = require('url');
+const { getURLFromFilePath } = require('internal/url');
+
+const { ModuleWrap, getNamespaceOfModuleWrap } = require('internal/loader/ModuleWrap');
+
+const ModuleMap = require('internal/loader/ModuleMap');
+const ModuleJob = require('internal/loader/ModuleJob');
+const resolveRequestUrl = require('internal/loader/resolveRequestUrl');
+const { Error } = global;
+const errors = require('internal/errors');
+
+function getBase() {
+  try {
+    return getURLFromFilePath(`${process.cwd()}/`);
+  } catch (e) {
+    e.stack;
+    // If the current working directory no longer exists.
+    if (e.code === 'ENOENT') {
```

Comment:
I don't think so, since it would still exist.

matches: https://github.com/nodejs/node/blob/244ada3c71588f02c10c0fcc526863660d460c96/lib/module.js#L664